app/src/main/res/drawable/ic_check_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I don't think we need to introduce a new drawable - you should be able to use `mozac_ic_check`